<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tetrixeasywindow</class>
 <widget class="QWidget" name="Tetrixeasywindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="nextPieceLabel">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>50</y>
     <width>81</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 5px double black;
</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="next">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>0</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>下一个方块</string>
   </property>
  </widget>
  <widget class="TetrixBoard" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>241</width>
     <height>483</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="quitButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>560</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLabel" name="image">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>601</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>170</y>
     <width>119</width>
     <height>75</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>得分</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="scoreLcd">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>251</y>
     <width>119</width>
     <height>76</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>333</y>
     <width>119</width>
     <height>75</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>消除行数</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="linesLcd">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>414</y>
     <width>119</width>
     <height>75</height>
    </rect>
   </property>
   <property name="digitCount">
    <number>4</number>
   </property>
  </widget>
  <widget class="QPushButton" name="loadButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>530</y>
     <width>85</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>加载</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>199</x>
     <y>530</y>
     <width>85</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>530</y>
     <width>85</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pauseButton">
   <property name="geometry">
    <rect>
     <x>107</x>
     <y>530</y>
     <width>86</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>暂停</string>
   </property>
  </widget>
  <zorder>image</zorder>
  <zorder>nextPieceLabel</zorder>
  <zorder>next</zorder>
  <zorder>frame</zorder>
  <zorder>quitButton</zorder>
  <zorder>label_2</zorder>
  <zorder>scoreLcd</zorder>
  <zorder>label_3</zorder>
  <zorder>linesLcd</zorder>
  <zorder>loadButton</zorder>
  <zorder>saveButton</zorder>
  <zorder>startButton</zorder>
  <zorder>pauseButton</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TetrixBoard</class>
   <extends>QFrame</extends>
   <header location="global">tetrixboard.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
